packages/docusaurus/src/client/exports/router.js

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
Are we gonna document this as a user-facing module?

Reply:
Probably few of them. I am afraid someone will use BrowserRouter inside pages

Reply:
Lets put it as hidden feature for now. I think matchPath is fine, and maybe react router hooks like useLocations